major_event_log/views.py

```diff
@@ -1,14 +1,26 @@
 """Defines all the views, which load the requested pages.
 
-These views will call the render the appropriate web pages based on
+These views will render the appropriate web pages based on
 templates defined in the 'templates/major-event-log' directory.
 """
+import uuid
+
+from django.shortcuts import render, get_object_or_404
+from django.core.urlresolvers import reverse
 from django.http import Http404
-from django.shortcuts import render
 
 from .models import Event
 
 
+def get_event_or_404(event_id):
+    """Retrieves event, if possible. If not, raises http 404 response."""
```

Comment:
I know you have already changed this once, but can we provide a more detailed description of what this is doing, why we need it?